### PR TITLE
feat:pipeline Random float & int modifier (#3)

### PR DESCRIPTION
**Pull request by DunDun1124**

### CODE REVIEW

File: src/core/pipeline/items/number/generation/random_float.rs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            let mut rng = thread_rng();
+            let random_number = rng.gen_range(start..end);
+            Ok(ctx.with_value(Value::F64(random_number)))
+        } else {
+            Err(ctx.with_invalid(format!("Value must be range")))
+        }
+    }
+}
+
+#[test]
```
Comment (victorteokw) [inline src/core/pipeline/items/number/generation/random_float.rs:38]: Hi @DunDun1124 we don't need to test someone else's function

File: src/core/pipeline/items/number/generation/random_int.rs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+#[async_trait]
+impl Item for RandomIntItem {
+    async fn call<'a>(&self, ctx: Ctx<'a>) -> Result<Ctx<'a>> {
+        let argument = self.argument.resolve(ctx.clone()).await?;
+        if argument.is_range() {
+            let r = self.argument.as_range().unwrap();
+            let start = r.start.resolve(ctx.clone()).await?.as_i32().unwrap();
+            let end = r.end.resolve(ctx.clone()).await?.as_i32().unwrap();
+            let mut rng = thread_rng();
```
Comment (victorteokw) [inline src/core/pipeline/items/number/generation/random_int.rs:29]: close range and open range is not handled

File: src/parser/std/pipeline/number/mod.rs [hunk 3]
```diff
@@ -11,3 +11,5 @@ pub(crate) fn is_even(_args: Vec<Argument>) -> Arc<dyn Item> {
 pub(crate) fn is_odd(_args: Vec<Argument>) -> Arc<dyn Item> {
     Arc::new(IsOddItem::new())
 }
+
+pub(crate) mod generation;
```
Comment (victorteokw) [inline src/parser/std/pipeline/number/mod.rs:15]: Let's put this line on top of the file.

**Comment by victorteokw** [comment]
HI @DunDun1124, let's remove the fmt commits. We will discuss fmt later.

**Comment by DunDun1124** [comment]
> HI @DunDun1124, let's remove the fmt commits. We will discuss fmt later.

Thank you for your review,I will fix it later.

**Pull request closed without merging**